bin/build-and-submit.sh

```diff
@@ -2,12 +2,11 @@
 # Build job and submit to Flink
 
 # Version of the jar file.
-# TODO: Extract from `build.gradle`.
 VERSION=$(./gradlew printVersion | head -1)
 JARFILE="cratedb-flink-jobs-${VERSION}.jar"
 
 # Build job
-./gradlew build
+./gradlew shadowJar
```

Comment:
no need, `build` calls the `shadowJar`

Reply:
But `build` does a lot of other things, and takes ages. Do you still want to enable it to improve coverage?

Reply:
yeah, I would keep build to be safe, in case we also add tests in the future.